internal/api/utils.go

```diff
@@ -44,3 +46,12 @@ func unwrapOrDefault[T any](value *T, defaultValue T) T {
 	}
 	return *value
 }
+
+func GenerateToken() string {
```

Comment:
Picking up from the naming remark from before:
```suggestion
func GenerateTransferToken() string {
```
Also, please add a docstring to explain the usage context.

Reply:
Please check the docstring that I added to the function.